crates/eframe/src/native/epi_integration.rs

```diff
@@ -278,13 +278,23 @@ impl EpiIntegration {
 
         let full_output = self.egui_ctx.run(raw_input, |egui_ctx| {
             if let Some(viewport_ui_cb) = viewport_ui_cb {
-                // Child viewport
+                // Child Deferred Viewport
                 crate::profile_scope!("viewport_callback");
                 viewport_ui_cb(egui_ctx);
             } else {
+                // ROOT Viewport ( with Immediate Viewport )
                 crate::profile_scope!("App::update");
                 app.update(egui_ctx, &mut self.frame);
             }
+
+            let continuous_mode = egui_ctx.options(|options| options.continuous_mode);
+            if continuous_mode {
+                let viewport_id = egui_ctx.viewport_id();
+                // TODO : Do not recall until the next repaint.
```

Comment:
The contributing guidelines say:

> Add your github handle to the TODO:s you write, e.g: `TODO(emilk): clean this up`

Reply:
> The contributing guidelines say:
> 
> > Add your github handle to the TODO:s you write, e.g: `TODO(emilk): clean this up`

thank you.

Reply:
> The contributing guidelines say:
> 
> > Add your github handle to the TODO:s you write, e.g: `TODO(emilk): clean this up`

If you know how to pass the doc test, please let me know.
It doesn't work with the "no_run" option.